### PR TITLE
Setup doc-warden to scan READMEs.

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR will setup doc-warden to run against the READMEs in the repo.

### CODE REVIEW

File: eng/pipelines/templates/jobs/archetype-sdk-client.yml [hunk 1]
```diff
@@ -50,6 +50,12 @@ jobs:
           swiftlint lint --path sdk/${{parameters.ServiceDirectory}} --quiet --reporter emoji
         displayName: 'Lint framework sources'
 
+      - script: |
```
Comment (bsiegel) [inline eng/pipelines/templates/jobs/archetype-sdk-client.yml]: We need to re-introduce the "Use Python Version" task before we run doc-warden

**Comment by mitchdenny** [comment]
Waiting on this PR to merge: https://github.com/Azure/azure-sdk-tools/pull/320

**Pull request merged**